docs/en/spark/configuration/transform-plugins/Split.md

```diff
@@ -4,19 +4,19 @@
 
 ## Description
 
-Split string according to `delimiter`
+Split string according to `separator`
```

Comment:
the option `delimiter` in `flink-split-transform` are is named `separator`, so i unify them.